config/config.yaml

```diff
@@ -39,6 +39,8 @@ branch-protection:
             - pull-cert-manager-deps
             - pull-cert-manager-chart
             - pull-cert-manager-e2e-v1-15
+            - pull-cert-manager-csi
+            - pull-cert-manager-csi-e2e-v16-v11
```

Comment:
`v16-v11`? I guess this is meant to be `v{kube_minor}-v{cm_minor}`, but it isn't really intuitive IMO.

I think `pull-cert-manager-csi-e2e-v1-16` is fine - the presubmit here needn't really care much about the CM version. We can create periodic jobs for testing the matrix of CM versions and include the CM version number in the name there. Storing two versions creates confusion and long names 😓